lab26/src/header/header.js

```diff
@@ -5,6 +5,19 @@ class Header extends React.Component {
   render() {
     return(
       <h1>RESTy</h1>
+      <nav>
```

Comment:
This is causing some issues, you need to have all of this wrapped in a JSX fragment, like this:
 <>....(all stuff being returned)...</>